tests/test_kharcha.py

```diff
@@ -13,8 +13,6 @@ class TestKharchaAPI(unittest.TestCase):
     def setUp(self):
         self.ctx = app.test_request_context()
         self.ctx.push()
-        db.drop_all()
-        db.create_all()
```

Comment:
@jace the next line calls `init_data()` which does both `drop_all()` and `create_all()`